### ISSUE TITLE
Starter kit DRS does not use https or port 443

### ISSUE THREAD

**Issue by ianfore** [ga4gh/ga4gh-starter-kit-drs#71]
The starter kit use of port 5000 for DRS servers does not conform to the following in the DRS 1.2 spec:

> The protocol is always https and the port is always the standard 443 SSL port. It is invalid to include a different port in a DRS hostname-based URI.

Acknowledging that there is a practical problem here, that for starter/demo/tutorial purposes the implementation https may distract from learning about DRS itself. Perhaps the DRS spec should allow for this usage of a port other than 443.

**Comment by yash-puligundla** [comment]
yes, that makes sense. We will update the documentation with this note. Should we also bring this up in the DRS specification repository?
